kubernetes/structures.go

```diff
@@ -310,8 +310,8 @@ func resourceListEquals(x, y api.ResourceList) bool {
 	return true
 }
 
-func expandLimitRangeSpec(s []interface{}, isNew bool) (api.LimitRangeSpec, error) {
-	out := api.LimitRangeSpec{}
+func expandLimitRangeSpec(s []interface{}, isNew bool) (*api.LimitRangeSpec, error) {
+	out := &api.LimitRangeSpec{}
```

Comment:
I don't think this needs changing - just an FYI - in Azureland we define these objects normally and then return the reference in the return e.g.

```
out := api.LimitRangeSpec{}
# ...
return &out
```

Reply:
Yeah, I had that inner dilemma myself.
Went for this approach to declutter a few `&` out of the code. I guess it's fine either way.